server/apps/distributed-app/docker-compose.yml

```diff
@@ -44,7 +44,7 @@ services:
       - "15672:15672"
 
   s3:
-    image: zenko/cloudserver:8.2.6
+    image: zenko/cloudserver:latest-7.10.19
```

Comment:
Hm are you sure about this one? we go from 8.2.6 to 7.10.19... Can you double check? It could be fixes on the v7 and we would just go one version back actually

Reply:
you are right, 

them hasn't stored all images on dockerhub
It should be `registry.scality.com/cloudserver/cloudserver:8.7.25`

tldr: https://github.com/scality/cloudserver/issues/4986

Reply:
Means they migrated to their own repo. As long as it's public, there is no issue switching to it IMO